develop/parachain-devs/runtime-development/FRAME/frame-overview.md

```diff
@@ -0,0 +1,110 @@
+---
+title: FRAME Overview
+description: FRAME is a powerful toolkit for Polkadot SDK runtime development, offering modular pallets and support libraries. Learn how to build your runtime logic.
+---
+
+# FRAME Overview
+
+## Introduction
+
+The runtime in a Polkadot SDK-based blockchain acts as the core component, encapsulating all essential business logic for:
+
+- Executing transactions
+- Managing state transitions
+- Facilitating node interactions
```

Comment:
Yeah, something like that, but all of this can be summarized as the runtime = state transition function. I agree`Facilitating node interactions` seems weird. I'll change it